docs/manual/introduction.tex

```diff
@@ -792,6 +792,10 @@
 Print information about the git repository from which the Checker Framework
 was compiled.
 
+\item
+ \<-AdumpOnErrors>
```

Comment:
Merge this with the `amount of detail` section earlier in this document and then add the details in `\ref{creating-debugging-options-detail}`.

Reply:
I have added the documentation in their corresponding sections in my latest commit